### PR TITLE
AWS Cloud Watch logs - fix proxy issue

### PR DESCRIPTION
**Pull request by Enigmatyk**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Fixed an issue where the proxy setting was not present.

## Description
Fixed an issue where the proxy setting was not present.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS-CloudWatchLogs/Integrations/AWS-CloudWatchLogs/AWS-CloudWatchLogs.yml [hunk 1]
```diff
@@ -65,6 +65,10 @@ configuration:
   section: Connect
   advanced: true
   required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
```
Comment (merit-maita) [inline Packs/AWS-CloudWatchLogs/Integrations/AWS-CloudWatchLogs/AWS-CloudWatchLogs.yml:71]: it's a needed field
```suggestion
  required: false
  section: Connect
```

File: Packs/AWS-CloudWatchLogs/ReleaseNotes/1_2_19.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### AWS - CloudWatchLogs
+
+- Fixed an issue where the proxy setting was not present.
```
Comment (merit-maita) [inline Packs/AWS-CloudWatchLogs/ReleaseNotes/1_2_19.md:6]:
```suggestion
- Added the *Use system proxy settings* parameter. 
- Updated the Docker image to *demisto/boto3py3:1.0.0.87887*.
```

**Review by merit-maita — CHANGES_REQUESTED** [review]
thank you for your contribution!
it looks good, please see my comments.

File: Packs/AWS-CloudWatchLogs/Integrations/AWS-CloudWatchLogs/AWS-CloudWatchLogs.yml (hunk 1 above)
Comment (Enigmatyk) [reply Packs/AWS-CloudWatchLogs/Integrations/AWS-CloudWatchLogs/AWS-CloudWatchLogs.yml:71]: Done.

File: Packs/AWS-CloudWatchLogs/ReleaseNotes/1_2_19.md (hunk 2 above)
Comment (Enigmatyk) [reply Packs/AWS-CloudWatchLogs/ReleaseNotes/1_2_19.md]: Done.

**Review by merit-maita — CHANGES_REQUESTED** [review]
as seen in the pr checks, there's a conflict
a new version of the pack was released very recently, with the same version as yours, so please change your version to be 1_2_20.md and in pack_metadata file as well.
thank you!

**Review by merit-maita — APPROVED** [review]

**Pull request merged**